docs/raising-events.md

```diff
@@ -95,17 +96,33 @@ forwarded to another implementation (wrapped object or base class) that
 FakeItEasy has no control over, so the fake doesn't know about the handlers and
 cannot call them.
 
-To work around this limitation, you will need to explicitly configure the calls
-to the event accessors to handle event subscription yourself, as in the example
-below:
+Similarly, strict fakes don't handle any call unless explicitly configured,
+including event subscription or unsubscription, so FakeItEasy also can't raise
+events on strict fakes.
+
+To work around this limitation, you have three options:
+
+- For a strict fake, you can enable the default event behavior on the fake at
+creation time, as explained [here](strict-fakes.md#events).

```

Comment:
nit: I'm not sure we SEO matters, but I think it's better to have the anchor text be something specific.
```suggestion
For a strict fake, you can [enable the default event behavior on the fake at
creation time](strict-fakes.md#events).
```

Reply:
Aaaaand now that I'm reading this, I wonder if the "wrappers" or calls base methods" folks will want a flag!
We don't have to do now. (Or at all.)